lib/babe/errors.go

```diff
@@ -72,6 +72,7 @@ var (
 	errNoBABEAuthorityKeyProvided = errors.New("cannot create BABE service as authority; no keypair provided")
 	errLastDigestItemNotSeal      = errors.New("last digest item is not seal")
 	errLaggingSlot                = errors.New("current slot is smaller than slot of best block")
+	errNoDigest                   = errors.New("no digest provided")
```

Comment:
nit It ultimately gets returned by `VerifyBlock` which is exported, so we *should* export this error

Reply:
> so we should export this error

When it is needed outside this package, not now

Reply:
I get your point, although then it defeats the purpose of having a sentinel error, we might as well just have a string with `fmt.Errorf` without a sentinel error. I guess then it's up for debate which is better, but having an unexported sentinel error is just wasteful in this case.

Reply:
`errNoDigest` gets used at multiple places, that's purpose here.